src/App.js

```diff
@@ -1,22 +1,57 @@
+import { useState, useEffect } from 'react';
+
 import './App.css';
 
 import ApplicatioHeader from './components/Layout/ApplicationHeader';
 import ApplicationBody from './components/Layout/ApplicationBody';
 import ApplicationFooter from './components/Layout/ApplicationFooter';
 
+import Modal from './components/Modal/Modal';
+import ModalContext from './contexts/modal-context';
 
-const App = () => (
-  <div className="app-layout">
-    <div className="app-layout-header">
-      <ApplicatioHeader titleLabel={'ReactJS Course'}></ApplicatioHeader>
-    </div>
-    <div className="app-layout-body">
-      <ApplicationBody></ApplicationBody>
-    </div>
-    <div className="app-layout-footer">
-      <ApplicationFooter></ApplicationFooter>
+
+const App = () => {
+  useEffect(() => { document.title = 'Mikalai Marozau | ReactJS Course' }, []);
+
+  // modal handlers
+  const [isModalActive, setIsModalActive] = useState(false);
```

Comment:
I think better solution will be to have only one state for modal which contain `modalData` and `active` fields

Reply:
Yeah. I kept it as separate state, trying to make check "if active" described above to work.

Reply:
Done